### PR TITLE
Iter1 1 login feature es

### PR DESCRIPTION
**Pull request by evansheng**
Completed cucumber steps for iteration 1-1
Login feature and create user feature

### CODE REVIEW

File: features/login.feature [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Background: user already has an account
+  
+  Given the following accounts exist:
+    | email                     | password          | admin |
+    | testuser@gmail.com        | password          | 0     |
+  
+Scenario: Not logged in user will get redirected to login page
+  Given I am not logged in
+  Given I visit the home page
+  Then I should see the login page
```
Comment (yashleya) [inline features/login.feature]: Page checks should say "Then I should BE ON the ______ page" instead of "see".  See can be used for view checks.

**Review by yashleya — COMMENTED** [review]
Change all page checks in cucumber steps

**Review by atancoder — APPROVED** [review]
LGTM

**Comment by atancoder** [comment]
Merging cucumber to master

**Comment by atancoder** [comment]
Done

**Pull request merged**